src/tests/testthat/testLoading.R

```diff
@@ -0,0 +1,19 @@
+library(methods)
+testthat::context("basic")
+
+testthat::test_that("can load digest from welcome page", {
+    wd$navigate(appURL)
+
+    loadButton <- wd$findElement("css", "#welcomeRequestDigestUpload")
+    loadButton$clickElement()
+
+    modal <- wd$findElement("css", ".modal-dialog")
+    waitForVisible(modal)
+
+    uploadDigestFile(wd)
+
+    panelTitle <- wd$findElement("css", inActivePane(".panelTitle"))
+    waitForVisible(panelTitle)
+    expectTextEqual("Upload spectrum file(s)", panelTitle)
```

Comment:
It would be good to check that at least some aspect of this save file - e.g. the notes, or the country - has been restored.